graphite/error.go

```diff
@@ -0,0 +1,46 @@
+package graphite // import "bosun.org/graphite"
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+func NewParseError(message string, err error, url url.URL) error {
+	if err != nil {
+		message += ": " + err.Error()
+	}
+	return fmt.Errorf("graphite ParseError: %s\nURL: %s\n", message, url.String())
+}
+
+func NewRequestError(message string, err error, url url.URL, resp *http.Response) error {
+	if err != nil {
+		message += ": " + err.Error()
+	}
```

Comment:
the whole body is an entire html document with body tags, a h2, some divs that override colors, etc.